ansible-101/Dockerfile.ssh

```
@@ -1,4 +1,4 @@
-FROM debian:stable-slim
+FROM debian:stable
```

```suggestion
FROM debian:bullseye
```

@DomenicoCutrupi can you confirm that the following works?